app/editor/src/features/admin/users/UserForm.tsx

```diff
@@ -40,7 +41,7 @@ export const UserForm: React.FC = () => {
     lookups.roles.map((r) => new OptionItem(r.name, r.id)),
   );
 
-  const isLinkedToKeycloak = user.key !== '00000000-0000-0000-0000-000000000000';
```

Comment:
This logic was to protect users that existed in Keycloak.  The reality is that in the future we won't be able to edit users in keycloak anyway.  So I'm wondering if there is any value in making this change.  For now you can simply comment this line of code out, as something similar will need to be added back.

Reply:
Sounds good!